Content.Server/Chemistry/EntitySystems/InjectorSystem.cs

```diff
@@ -34,10 +36,10 @@ private bool TryUseInjector(Entity<InjectorComponent> injector, EntityUid target
         // Handle injecting/drawing for solutions
```

Comment:
Might want to keep the whole result in a variable, so that it doesn't query OpenableSystem on every Solution component if the thing's closed, but just does it once.

```suggestion
        var openOrIgnored = injector.Comp.IgnoreClosed || !_openable.IsClosed(target);
        // Handle injecting/drawing for solutions
```